client/multi-currency-setup/tasks/add-currencies-task/index.js

```diff
@@ -0,0 +1,339 @@
+					: sprintf(
+							_n(
+								'Add %s currency',
+								'Add %s currencies',
+								activatedCount,
+								'woocommerce-payments'
+							),
+							activatedCount
+					  ) }
+			</Button>
+		);
+	};
+
+	const displayCurrencyCheckbox = ( code ) =>
+		hiddenCurrencies.includes( code ) ? null : (
+			<EnabledCurrenciesModalCheckbox
+				key={ 'currency-checkbox-' + availableCurrencies[ code ].id }
+				checked={ selectedCurrencies[ code ] }
+				onChange={ handleChange }
+				currency={ availableCurrencies[ code ] }
+			/>
+		);
+
+	return (
+		<WizardTaskItem
+			className="add-currencies-task"
+			title={ __( 'Add currencies', 'woocommerce-payments' ) }
+			visibleDescription={ sprintf(
+				_n(
+					'%s currency added',
+					'%s currencies added',
+					activatedCount,
+					'woocommerce-payments'
+				),
+				10 > activatedCount
+					? _.capitalize( numberWords[ activatedCount ] )
+					: activatedCount
+			) }
+			index={ 1 }
+		>
+			<CollapsibleBody>
+				<p className="wcpay-wizard-task__description-element is-muted-color">
+					{ ! isLoading &&
+						interpolateComponents( {
+							mixedString: __(
+								"Add currencies so international customers can shop and pay in their local currency. Your store's " +
+									'default currency is {{storeCurrencyText /}}.',
+								'woocommerce-payments'
+							),
+							components: {
+								storeCurrencyText: (
+									<strong>
+										{ StringRepresentationOfCurrency(
+											defaultCurrency
+										) }
+									</strong>
+								),
+							},
+						} ) }
+				</p>
+				{ 1 < enabledCurrenciesLength && (
+					<p className="wcpay-wizard-task__description-element is-muted-color">
+						{ interpolateComponents( {
+							mixedString: __(
+								"You've already added {{enabledCurrenciesText /}} to your store.",
+								'woocommerce-payments'
+							),
+							components: {
+								enabledCurrenciesText: (
+									<strong>
+										{ ConcatenateCurrencyStrings(
+											enabledCurrencyCodes,
+											defaultCurrencyCode,
+											availableCurrencies
+										) }
+									</strong>
+								),
+							},
+						} ) }
+					</p>
+				) }
+				<Card className="add-currencies-task__currency-selector-wrapper">
+					<CardBody>
+						<div className="add-currencies-task__search">
+							<Search
+								value={ searchText }
+								placeholder={ __(
+									'Search currencies',
+									'woocommerce-payments'
+								) }
+								onChange={ handleSearchChange }
+							/>
+						</div>
+						<div className={ 'add-currencies-task__separator' }>
+							&nbsp;
+						</div>
+						{ searchText ? (
+							/* translators: %1: filtered currencies count */
+							<h4>
+								{ sprintf(
+									__(
+										'Search results (%1$d currencies)',
+										'woocommerce-payments'
+									),
+									filteredCurrencyCodes.length
+								) }
+							</h4>
+						) : (
+							''
+						) }
```

Comment:
Nitpick here. In JSX you can writ this like `searchtext && ()` rathern than `searchText ? () : ()`.
There are a couple of other cases in this file.